Snakefile

```diff
@@ -575,10 +576,12 @@ rule post_masked_consensus_and_summary_stats_to_id3c:
             except HTTPError as http_err:
                 file.write(str(http_err))
 
+            raise http_err from None
+
```

Comment:
The `from None` clause is not doing anything in this case because the `raise` isn't happening in a nested exception handler: it's in the first `except` for the POST to ID3C, not the second `except` for the POST to Slack. If it was in the latter, the `from None` would suppress reporting of the ID3C exception and just report the Slack exception. I've used this from in various places in ID3C when I want to convert from exception A to exception B without reporting extra confusing details about A.

Additionally, there's also some variable shadowing going on because both `except` blocks use the same variable name for the exception. This means that by the time the `raise http_err from None` line is reached, `http_err` is actually undefined, which produces a NameError. The NameError still causes the build to fail, but not for the reason we wanted! :-) and not with the details of the HTTP exception. This can be fixed by using different variable names for the two exceptions.

Reply:
Fixed it! Thanks, Thomas.